scripts/generate-kubescape-exceptions.py

```
@@ -0,0 +1,229 @@
+        attributes = {"kind": anchor(entry["kind"], path, name)}
+        if "name" in entry:
+            attributes["name"] = anchor(entry["name"], path, name)
+        if "namespace" in entry:
+            attributes["namespace"] = anchor(entry["namespace"], path, name)
+        designators.append(
+            {"designatorType": "Attributes", "attributes": attributes}
+        )
+    return designators
+
+
+def convert_document(doc, path):
```

Ruff (0.15.20)

[warning] Too many branches (14 > 12)

(PLR0912)